gnovm/pkg/gnolang/values_conversions.go

```diff
@@ -44,6 +44,7 @@ func ConvertTo(alloc *Allocator, store Store, tv *TypedValue, t Type) {
 			tv.T = t
 			return
 		} else {
+			// both NativeType, use reflect to assert.
```

Comment:
Is this comment correct? It looks as though `t` is not a native type in this block.

Reply:
reverted.